README.md

```diff
@@ -11,69 +11,97 @@ git clone https://github.com/Police-Data-Accessibility-Project/data-sources-app.
 cd data-sources-app
 ```
 
-### 2. Create an .env file in the root directory.
-
-It should have a SUPABASE_URL and SUPABASE_KEY for PDAP's Data Sources [Supabase](https://supabase.com/).
-
-### 3. Create a virtual environment.
+### 2. Create a virtual environment.
 
 If you don't already have virtualenv, install the package:
 
 ```
+
 pip install virtualenv
+
 ```
 
 Then run the following command to create a virtual environment:
 
 ```
+
 virtualenv -p python3.9 venv
+
 ```
 
-### 4. Activate the virtual environment.
+### 3. Activate the virtual environment.
 
 ```
+
 source venv/bin/activate
+
 ```
 
-### 5. Install dependencies.
+### 4. Install dependencies.
 
 ```
+
 pip install -r requirements.txt
+
+```
+
+### 5. Manually export a SUPABASE_URL and SUPABASE_KEY in the command line
+
+The app should have a SUPABASE_URL and SUPABASE_KEY for PDAP's Data Sources [Supabase](https://supabase.com/).
+
+```
+export SUPABASE_URL="**"
+export SUPABASE_KEY="**"
+
```

Comment:
can you add the psycopg2 url format in here as well? Both is fine for now, we will cut the data sources end point over soon and remove the Supabase creds